pkg/database/import.go

```diff
@@ -0,0 +1,88 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/rotisserie/eris"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+// Import will copy the contents of input db to output db.
+func Import(input, output *DbInstance, dryRun bool) error {
+
+	in := input.DB
+	if err := output.DB.Transaction(func(out *gorm.DB) error {
+
+		if err := importTable(in, out, dryRun, Experiment{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, ExperimentTag{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Run{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Tag{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Metric{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, LatestMetric{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Param{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Dashboard{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, App{}); err != nil {
+			return err
+		}
+
+		return nil
+	}); err != nil {
+		return eris.Wrapf(err, "error importing table data")
+	}
+
+	fmt.Println("Data transfer complete")
+	return nil
+}
+
+// importTable will copy the contents of one table (model) from sourceDB to destDB.
+func importTable[T any](sourceDB, destDB *gorm.DB, dryRun bool, model T) error {
+
+	// Query data from the source database
+	var sourceData []T
+	fmt.Printf("Importing %T\n", sourceData)
+	tx := sourceDB.Model(&model).Find(&sourceData)
```

Comment:
we probably want to use `Rows` to be able to stream from input to output

Reply:
done